src/Corax/Analyzers/Analyzer.cs

```diff
@@ -75,6 +76,8 @@ public static ArrayPool<Token> TokensPool
             float tokenBufferMultiplier = 1;
             foreach( var transformer in transformers)
             {
+                IsExactAnalyzer = transformer is ExactTransformer;
```

Comment:
this sets that according to last transformer. Is this correct?

Reply:
We only use 1 right now, but you are correct WRT future. Fixed.